src/Export/MongoDBTarget.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * @file
+ * Contains MongoDB implementation of ExtractionTargetInterface.
+ */
+
+namespace App\Export;
+
+use App\Document\Entry;
+use App\Document\ExtractionResult;
+use App\Repository\EntryRepository;
+use App\Repository\ExtractionResultRepository;
+use Doctrine\ODM\MongoDB\DocumentManager;
+
+/**
+ * Class MongoDBTarget.
+ */
+class MongoDBTarget implements ExtractionTargetInterface
+{
+    private $documentManager;
+    private $extractionResultRepository;
+    private $entryRepository;
+    /* @var array $types */
+    private $types;
+
+    /**
+     * MongoDBTarget constructor.
+     *
+     * @param \Doctrine\ODM\MongoDB\DocumentManager $documentManager
+     * @param \App\Repository\EntryRepository $entryRepository
+     * @param \App\Repository\ExtractionResultRepository $extractionResultRepository
+     */
+    public function __construct(DocumentManager $documentManager, EntryRepository $entryRepository, ExtractionResultRepository $extractionResultRepository)
+    {
+        $this->documentManager = $documentManager;
+        $this->extractionResultRepository = $extractionResultRepository;
+        $this->entryRepository = $entryRepository;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function initialize(): void
+    {
```

Comment:
Would be nice with comment why these are empty in this target?

Reply:
added